src/index.js

```diff
@@ -57,15 +54,17 @@ module.exports.workspaceActions = [
     action: async (context, models) => {
       try{
```

Comment:
just to keep consistency:
```suggestion
      try {
```